droid-system.inc

```diff
@@ -21,7 +21,11 @@ Summary:    System%{variant} package for Droid HAL adaptations
 Version:    1
 Release:    1
 Group:      Development/Tools
+%if 0%{?system_license:1}
```

Comment:
Wouldn't it be possible to determine if license was set before or force the set of the license explicitly in the template?
The later seems to be more clean IMHO.

Reply:
Unfortunately not straightforward due to how RPM macro pre-processor works.
Also even if there was a way to tap into macro expansion, there is no %{license} macro (there is %license section, but that's different).
When trying to force License the second time, RPM errors out with
`error: Duplicate License entries in package: (main package)`

Reply:
It could be possible in lua. In any case I would prefer the license to be set explicitly or assumed from what AOSPs license is.

Reply:
We use macros as switches everywhere, patches rewritten in lua welcome if readability won't suffer:)
https://github.com/mer-hybris/droid-hal-version/blob/8f869d7aa4e7d541f14c63afed71589d45939197/droid-hal-version.inc#L56
https://github.com/mer-hybris/droid-hal-device/blob/master/droid-hal-device.inc#L165

AOSP license will be set here:
https://github.com/mer-hybris/droid-system-sony-template/blob/aosp-license/droid-system-license.inc

Reply:
Lets not complicate this with lua. :)